resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesConf.scala

```diff
@@ -55,6 +55,9 @@ private[spark] abstract class KubernetesConf(val sparkConf: SparkConf) {
       }
   }
 
+  def workerDecommissioning: Boolean =
```

Comment:
I'd avoid adding getters for simple configs.

Reply:
Is that a hard preference? I find this easier to write.

Reply:
I prefer to avoid these, but don't care that much.